challenges.tex

```diff
@@ -0,0 +1,540 @@
+and RS1 is placed at $\texttt{M\_Z25}$.}
+	\label{fig:grasping-field}
+\end{figure}
+
+\subsubsection{Product Challenges}\label{sec:challenge-cx}
+This section covers four types of challenges, which all can be individually
+completed.
+Each challenge corresponds to the production of a product with one of the
+available complexities (C0, C1, C2, C3) in the \ac{RCLL} using either
+one or two \ac{RS}.\\
+For complexities C1, C2 and C3 the accumulated cost for mounting the required
+rings must be equal to 1, 2 and 3, respectively.
+\textbf{Task:} Produce all posted orders.\\
+\begin{table}[!htb]
+ \centering
+ \begin{tabular}{l|l|l}
+  Machines & Challenge type & Scoring \\\hline
+  2 & C0 & 30 \\
+  3 & C1 & 50 \\
+  4 & C1 & 50 \\
+  3 & C2 & 70 \\
+  4 & C2 & 70 \\
+  3 & C3 & 100 \\
+  4 & C3 & 100 \\
+ \end{tabular}
+ \caption{CX Challenge}
+ \label{tab:challenge-cx}
+\end{table}
+
+\subsubsection{Exploration + Production Challenges}\label{sec:challenge-combine-exp-c0}
+The same challenge as in \refsec{sec:challenge-cx} but without receiving the
+machine positions. Hence exploration and production takes place concurrently.
+The same point scores as in \reffig{tab:challenge-cx} apply with an additional
+$+20$ per challenge.
+%\subsubsection{RefBox Simulation Challenge}\label{sec:challenge-simulation}
+%A competition on the agent level. The RefBox provides a set of actions
+%that can be executed by sending dedicated commands to the RefBox via protobuf.
+%Hence no actual robot is required to participate.
+%\textbf{Task:} Play a regular RCLL game through the RefBox simulation
+%interface.\\
+%Participating in this challenge yields points based on the achieved in-game
+%points, see \reftab{tab:challenge-simulation}.
+%Additionally, the team scoring the highest points overall gets awarded another
+%$10$ points
+%\begin{table}[!htb]
+% \centering
+% \begin{tabular}{l|l|l|l}
+%  Points & Scoring \\\hline
+%  $[0,50)$ & 0\\% chktex 9
+%  $[50,150)$ & 20\\% chktex 9
+%  $[150,250)$ & 40 \\% chktex 9
+%  $[250,\infty)$ & 60 \\% chktex 9
+% \end{tabular}
+% \caption{CX Challenge}
+% \label{tab:challenge-cx}
+%\end{table}
+
+
+\subsubsection{Markerless Detection Challenge}\label{sec:markerless}
+Image recognition challenge to classify different machine types.\\
+\textbf{Task:} Autonomously label the machines shown in a set of pictures
+by their type.
+A picture may contain more than one machine, which all count towards the
+total number of machines to detect.\\
+As a preparation for this challenge, a data set will be supplied to
+all participants which may be used for training and testing purposes
+\footnote{\url{https://fh-aachen.sciebo.de/s/MRghypMQSMhTYaC}}.
+The evaluation set for the challenge consists of a set of separate images.
+\begin{table}[!htb]
+ \centering
+ \begin{tabularx}{\linewidth}{l|l|l|l}
+  \% Correctly Classified & \% Wrongly Classified & \% Not Classified
+  & Scoring \\\hline
+  $x$ & $y$ & $z$ & $(x-y)\cdot30$
```

Comment:
So the maximum score is 3000 points? That's a lot

Reply:
Isn't it less, I think it not possible to distinguish Cyan from Magenta machines, and 1 and 2 (RS1 vs. RS2). Which leads to BS, CS, RS, DS, SS so i guess maximum is 5 * 30 = 150? But since producing a C3 is only worth 100 points, 120 if it is done without maschine positions. 150 is still a lot of points (most that any challange gives). Do we want to push teams to put effort into this over other challanges? If not I would suggest lowering the points, in my opinion other challanges are harder to achieve.

Reply:
I believe the number 30 as just an arbitrary number and I agree that it should be lower. I think a max of 50-70 points might be a good way to go.

Reply:
It was intended to bring 30 points total across all data to be labeled (independent of the types). I clarified that.